### ISSUE TITLE
Pull `create-react-app`

### ISSUE THREAD

**Issue by jakub-bao** [pepfar-datim/datim-mechanism-finder#2]
Git checkout `create-react-app`.
Install dependencies.
Try to run in dev mode.
Try to do production build.

https://www.npmjs.com/package/create-react-app
https://facebook.github.io/create-react-app/docs/getting-started

**Comment by jakub-bao** [comment]
Hi Tom @tomzemp ,

I got some time on the user management app so I'll hop on the mechanisms finder for a bit while I am waiting on more feedback on my project.

Let me grab this ticket and initialize an empty REACT project... 
I've create a new "master" branch for us: https://github.com/pepfar-datim/datim-mechanism-finder/tree/react

So feel free to jump in at any time.. just branch of the `react` branch and you can grab a ticket.
Also feel free to create a new ticket based on what you think needs to be done.
